tests/bin/expected/petstore.ts

```diff
@@ -194,6 +226,9 @@ export interface paths {
   }
   '/store/order': {
     post: {
+      requestBody: {
+        '*/*': components['schemas']['Order']
```

Comment:
Ah interesting. Didn’t even know you could do this

Reply:
I'm not exactly sure if this is part of the specification, but GitHub is using it for its ["Upload a release asset" endpoint](https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#upload-a-release-asset), which allows you to upload any kind of file, of any type